src/content/popup/names.ts

```diff
@@ -22,143 +17,26 @@ export function renderNamesEntries({
   more: boolean;
   options: ShowPopupOptions;
 }): HTMLElement {
-  const namesTable = html('div', { class: 'name-table entry-data' });
+  const containerElement = html('div', {
+    /* Make sure it's possible to scroll all the way to the bottom of each kanji
+     * table. */
+    style:
+      'scroll-snap-align: start; scroll-margin-bottom: var(--expand-button-allowance);',
+  });
```

Comment:
Copied from the kanji tab. Is this also needed in the names tab?

Reply:
Good question. I'd have to experiment I'm afraid. If it's not currently used in the names tab then I think it's safe to leave it off for now.

Reply:
Yeah, scroll snap is currently not used in the names tab at all, so there should be no reason to add this style.